src/JustEat.StatsD.Tests/JustEat.StatsD.Tests.xproj

```diff
@@ -7,7 +7,7 @@
   <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.Props" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup Label="Globals">
     <ProjectGuid>8d9aa1a3-2619-4800-aaec-84ae4da15455</ProjectGuid>
-    <RootNamespace>JustEat.StatsD</RootNamespace>
+    <RootNamespace>JustEat.StatsD.Tests</RootNamespace>
```

Comment:
I prefer to put the root namespace of the tests the same as the root of the thing under test, otherwise once you drop down a namespace in the tests you end up having to import namespaces all the time rather than having a one-to-one match.

Target: ```Foo.Bar``` -> ```Foo.Bar.Baz``` 
Tests : ```Foo.Bar``` -> ```Foo.Bar.Baz```

compared to:

Target: ```Foo.Bar``` -> ```Foo.Bar.Baz``` 
Tests: ```Foo.Bar.Tests``` (```Foo.Bar``` is implied) -> ```Foo.Bar.Tests.Baz``` and ```Foo.Bar.Baz```